Dockerfile

```diff
@@ -41,7 +41,12 @@ RUN \
   apt-get -qq update && \
   apt-get -qq install apt-utils 2>/dev/null && \
   apt-get -qq dist-upgrade && \
+  echo "applying patch for https://github.com/docker-mailserver/docker-mailserver/issues/2023" && \
+  mv /bin/hostname{,.bak} && \
+  echo "echo docker.mail.server" > /bin/hostname && \
```

Comment:
`.server` may not be a valid TLD presently, but it's not reserved either. Do not use this value. Use a [reserved TLD](https://en.wikipedia.org/wiki/Top-level_domain#Reserved_domains) that we know will not resolve to any domain in future (_even if that's not likely to be concern during build_).

I suggest `.invalid` as appropriate here. `dms.invalid` or `docker-mailserver.invalid` should be fine.